project/src/main/java/com/google/sps/Servlets/FolderServlet.java

```diff
@@ -25,13 +25,28 @@
 public class FolderServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
this function was approved in a previous pull request and I added it back exactly; feel free to just skim